netmiko/nokia/nokia_srl.py

```diff
@@ -50,7 +50,7 @@ def session_preparation(self) -> None:
             "environment complete-on-space false",
```

Comment:
Change with:
```
commands = [
            r"environment complete-on-space false",
            r"environment cli-engine type basic",
        ]
        for command in commands:
            self.disable_paging(command=command, cmd_verify=True, pattern=r"#")
```
Order of the command seems important because it was failing if environment complete-on-space false is executed last.

Reply:
Okay fixing...